src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -312,6 +328,14 @@ class WorkspaceMembersPage extends React.Component {
                         />
                     </View>
                 </View>
+                <FixedFooter
+                    style={[styles.pt3, styles.pb3]}
+                >
+                    <OfflineText
+                        outerContainerStyles={[styles.flexRow, styles.justifyContentCenter, styles.alignItemsCenter, styles.overflowHidden, styles.w100]}
+                        message={this.props.translate('workspace.people.offlineMessage')}
+                    />
```

Comment:
I understand that it is a good idea to create a generic component but this is not the correct PR to do so. Offline support is still in planning so lets just move the necessary UI parts here from  `OfflineText` .